backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxUtils.java

```diff
@@ -66,11 +67,16 @@ public static void addPageBreak(XWPFDocument document) {
     }
 
     public static void setText(XWPFRun run, String text) {
-        String[] split = text.split("\n");
-        run.setText(split[0]);
-        for (int i = 1; i < split.length; i++) {
-            run.addBreak();
-            run.setText(split[i]);
+        if (CommonUtils.isNullEmptyOrWhitespace(text)) {
+            run.setText("");
+            return;
```

Comment:
Maybe `return` statement can be removed , since it is end of execution for that method

Reply:
Removed it